modules/importer/src/model/mod.rs

```diff
@@ -239,11 +242,13 @@ impl TryFrom<Model> for Importer {
             progress_total,
             progress_message,
             continuation,
-            revision: _,
+            heartbeat,
+            ..
```

Comment:
The idea of this pattern is to be aware of fields which get added. By a compiler error. So I'd want to keep it.